src/main/resources/com.exclamationlabs.connid.base.zoom/scopeAccessList.csv

```diff
@@ -0,0 +1,24 @@
+account:master
```

Comment:
Is this .csv just for documentation purposes?  I'm ok with having this in there but you may want to refer to in README.MD if the intent is to document the required scopes.

Reply:
this info is a place holder for documentation purposes. Updating the readme in the next few days.